libs/zk/include/nil/crypto3/zk/snark/systems/plonk/placeholder/lookup_argument.hpp

```diff
@@ -419,7 +421,17 @@ namespace nil {
                                 math::polynomial_dfs<typename FieldType::value_type> v = (typename FieldType::value_type(t_id + 1)) * lookup_tag;
                                 theta_acc = theta;
                                 for (std::size_t i = 0; i < l_table.columns_number; i++) {
-                                    v += theta_acc * lookup_tag * plonk_columns.constant(l_table.lookup_options[o_id][i].index);
+                                    math::polynomial_dfs<typename FieldType::value_type> c;
+                                    // Not switch because of warnings
+                                    if( l_table.lookup_options[o_id][i].type == VariableType::column_type::witness )
```

Comment:
This will become a function inside the class of "plonk_columns". Something like "get_variable_value" or something that sounds better?